logic/src/commonJvmAndroid/kotlin/com/wire/kalium/logic/data/message/ProtoContentMapper.kt

```diff
@@ -17,6 +18,24 @@ class ProtoContentMapperImpl: ProtoContentMapper {
                     .build()
                 builder.text = text
             }
+            is MessageContent.DeleteMessage -> {
+                val deleted = Messages.MessageDelete.newBuilder()
+                    .setMessageId(messageContent.messageId)
+                    .build()
+                builder.deleted = deleted
+            }
+            is MessageContent.HideMessage -> {
+                val qualifiedConversationId = Messages.QualifiedConversationId.newBuilder()
+                    .setId(messageContent.conversationId.value)
+                    .setDomain(messageContent.conversationId.domain).build()
+                val hidden = Messages.MessageHide.newBuilder()
+                    .setMessageId(messageContent.messageId)
+                    //based on the documentation this conversation id is deprecated, but somehow it crashed when not passing it!
```

Comment:
It's deprecated because it's superseded by `QualifiedConversationId` but we can't remove it because:
1. it's marked as `required` in the protobuf definition and therefore it must always be included
2. We need be compatible with older clients